recipes/pyxb/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.2.5" %}
+
+package:
+  name: pyxb
+  version: {{ version }}
+
+source:
+  fn: PyXB-{{ version }}.tar.gz
+  url: https://github.com/pabigot/pyxb/archive/PyXB-{{ version }}.tar.gz
+  sha256: be218aac18573ad082afbbcb87230db863564f11779d881096c9dd228db7b14e
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  requires:
+    - nose
+  imports:
+    - pyxb
+  commands:
+    - cd ${SRC_DIR}/tests && nosetests *
```

Comment:
I think you need to use different `cd` syntax for windows and non-windows. See the following for an example:

https://github.com/conda-forge/staged-recipes/pull/1138/files#diff-359a956b2fb22cc654bc16d8e80cff8aR30